contrail/contrail-tempest-init.sh

```diff
@@ -95,17 +99,28 @@ iniset $TEMPEST_CONFIG compute ssh_user ${DEFAULT_INSTANCE_USER:-cirros}
 iniset $TEMPEST_CONFIG compute image_ref $image_uuid
 iniset $TEMPEST_CONFIG validation image_ssh_user ${DEFAULT_INSTANCE_USER:-cirros}
 iniset $TEMPEST_CONFIG validation image_ssh_password ${DEFAULT_INSTANCE_PASSWORD:-cubswin:)}
+iniset $TEMPEST_CONFIG validation run_validation true
+iniset $TEMPEST_CONFIG compute min_compute_nodes $compute_count
+
 iniset $TEMPEST_CONFIG compute flavor_ref $FLAVOR_ID
+iniset $TEMPEST_CONFIG compute flavor_ref_alt $FLAVOR_ID_ALT
 iniset $TEMPEST_CONFIG compute image_ref_alt $image_uuid_alt
 iniset $TEMPEST_CONFIG compute image_alt_ssh_user ${DEFAULT_INSTANCE_USER:-cirros}
 iniset $TEMPEST_CONFIG compute image_alt_ssh_password ${DEFAULT_INSTANCE_PASSWORD:-cubswin:)}
 iniset $TEMPEST_CONFIG compute allow_tenant_isolation ${TENANT_ISOLATION:-false}
 iniset $TEMPEST_CONFIG network public_network_id "$public_network_id"
+iniset $TEMPEST_CONFIG network floating_network_name $PUBLIC_NETWORK_NAME
+
+iniset $TEMPEST_CONFIG network-feature-enabled api_extensions allowed-address-pairs,extra_dhcp_opt,security-group,floating_ip,port_security,ipv6,router,quotas,binding
+iniset $TEMPEST_CONFIG compute-feature-enabled live_migration false
+iniset $TEMPEST_CONFIG compute-feature-enabled cold_migration false
+iniset $TEMPEST_CONFIG compute-feature-enabled scheduler_available_filters "RetryFilter, AvailabilityZoneFilter, RamFilter, DiskFilter, ComputeFilter, ComputeCapabilitiesFilter, ImagePropertiesFilter, ServerGroupAntiAffinityFilter, ServerGroupAffinityFilter"
+
+
+iniset $TEMPEST_CONFIG identity-feature-enabled api_v2 false
+iniset $TEMPEST_CONFIG identity-feature-enabled api_v2_admin false
+iniset $TEMPEST_CONFIG identity-feature-enabled api_v3 false
 
-# Disable IPv6 tests
-iniset $TEMPEST_CONFIG network-feature-enabled ipv6 true
-iniset $TEMPEST_CONFIG network-feature-enabled port_security false
-iniset $TEMPEST_CONFIG network-feature-enabled floating_ips false
```

Comment:
is floating_ips test disabled by default?

Reply:
There was a typo, i had set it as floating_ip instead of floating_ips
Added a patch : https://github.com/Juniper/tempest/pull/22/commits/c78e8517f3daf3925beb713f8e0bd89596d9d051